services/s3/src/test/java/software/amazon/awssdk/services/s3/internal/multipart/MultipartDownloadIntegrationTest.java

```diff
@@ -51,7 +49,7 @@ class MultipartDownloadIntegrationTest {
 
     private S3AsyncClient s3;
     private final SplittingTransformerConfiguration splitConfig = SplittingTransformerConfiguration.builder()
-                                                                                                   .bufferSize(1024 * 1024 * 32L)
+                                                                                                   .bufferSizeInBytes(1024 * 1024 * 32L)
```

Comment:
nit : Would improve readability if we had a variable denoting that this is the default value : 
```
private final int DEFAULT_BUFFERED_SIZE_IN_BYTES = 1024 * 1024 * 32L; 
```
With a comment that the default is 32MB.

Reply:
like mentioned in the other comment, please ignore this class